api/src/main/resources/workbench-api.yaml

```diff
@@ -8275,6 +8275,46 @@ definitions:
         description: |-
           If the institutional_role_enum is "other", custom description
           of this user's role in the institution.
+      ethnicity:
+        description: |-
+          User-reported ethnicity information
+        "$ref": "#/definitions/Ethnicity"
+      identifiesAsLgbtq:
+        description: |-
+          User-reported identities as lgbtq information
+        type: boolean
+      lgbtqIdentity:
+        description: |-
+          User-reported lgbtqIdentity information
+        type: string
+      yearOfBirth:
+        description: |-
+          User-reported yearOfBirth information
+        type: number
+      highestEducation:
+        description: |-
+          User-reported highestEducation information
+        "$ref": "#/definitions/Education"
+      disability:
+        description: |-
+          User-reported disability
+        "$ref": "#/definitions/Disability"
+      degrees:
+        description: |-
+          User-reported degrees
+        type: string
+      genderIdentity:
```

Comment:
I think any of these that may be repeated should be made plural, e.g. genderIdentities, races, ...